backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-backing-up-pvs-csi-doc.adoc

```diff
@@ -26,9 +26,14 @@ kind: VolumeSnapshotClass
 metadata:
   name: <volume_snapshot_class_name>
   labels:
-    velero.io/csi-volumesnapshot-class: "true"
+    velero.io/csi-volumesnapshot-class: "true" <1>
+  annotations:
+    snapshot.storage.kubernetes.io/is-default-class: true <2>
 driver: <csi_driver>
-deletionPolicy: Retain
+deletionPolicy: <Retain_or_Delete> <3>
 ----
+<1> Must be set to `true`.
+<2> Must be set to `true`.
+<3> OADP supports both `Retain` and `Delete` as a `deletionPolicy` for CSI VSC backup and restore. For the OADP 1.2 Data Mover set the `deletionPolicy` to `Retain`.
```

Comment:
We should add the CSI and DataMover in step no.3

Reply:
<3> OADP supports both `Retain` and `Delete` as a `deletionPolicy` for CSI VSC backup and restore, CSI backup and restore, and Data Mover. For the OADP 1.2 Data Mover set the `deletionPolicy` to `Retain`.

@PrasadJoshi12 Is this what you meant?